src/app/Containers/app.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
create containers for the main views of our application...

containers are the "smart" components that manager state
state is passed down to the "dumb" components through this.props